app/assets/javascripts/cloud/bootstrap.js

```diff
@@ -0,0 +1,64 @@
+$(function() {
+  // https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable-string
```

Comment:
Is it fine to reuse code without an explicit license?

Reply:
According to StackOverflow, that code is licenced CC-BY-SA; and has been properly attributed by the comment.

https://meta.stackexchange.com/questions/271080/the-mit-license-clarity-on-using-code-on-stack-overflow-and-stack-exchange

Reply:
Right, thank you.

Reply:
Just to be on the safe side, I've engaged our legal team to verify.

Reply:
I've added a second comment with the license; legal has approved.